API/testrunner/testrunner.py

```diff
@@ -141,7 +141,7 @@ def __run_test_on_device(self, app, device, testset, test):
         '''
         testfile = utils.join(device.get_test_path(), testset, test['name'])
 
-        if device.get_type() is 'stm32f4dis':
+        if device.get_type() is 'stm32f4dis' or device.get_type() is 'artik053':
```

Comment:
A shorter solution: `if device.get_type() in ['stm32f4dis', 'artik053']`

Reply:
Thanks for the review. I update.